src/CasingOfString.md

```diff
@@ -92,3 +93,33 @@ func CheckCase(input string) string {
 	}
 }
 ```
+
+### Python
+```python
+import unicodedata
+
+def check_case(input_str):
```

Comment:
function should match with example result above

Reply:
Fixed